libs/blocks/caas/utils.js

```diff
@@ -248,9 +248,12 @@ const getFilterArray = async (state) => {
   return filters;
 };
 
-const getCountryAndLang = ({ autoCountryLang, country, language }) => {
+export function getCountryAndLang(
+  { autoCountryLang, country, language },
+  miloConfig = pageConfig,
```

Comment:
Are you only using the `miloConfig` param for the tests? If so, that's not a great practice. What about using `setConfig` in the test instead?

Reply:
I think what you're really asking is for me to use `getConfig` in body of func and not the `pageConfig` global param that probably shouldn't exist. That's the only way a test with `setConfig` would work. The pattern I'm using here is the one that already exists in the code (see top of file). There was no test coverage for this func previously.

Reply:
Fixed at [23223b4](https://github.com/adobecom/milo/pull/945/commits/23223b44ae00cd76f6de9b031c00adb79f2fcc8c)